packages/eddsa-poseidon/src/eddsa-poseidon.ts

```diff
@@ -26,12 +31,15 @@ import { hash as blake, checkMessage, checkPrivateKey, isPoint, isSignature, pru
  * and converting it into a little-endian integer. The use of the secret scalar streamlines the public key generation
  * process by omitting steps 1, 2, and 3 as outlined in RFC 8032 section 5.1.5, enhancing circuit efficiency and simplicity.
  * This method is crucial for fixed-base scalar multiplication operations within the correspondent cryptographic circuit.
+ * The private key must be an instance of Buffer, Uint8Array or a string. If you want to pass a bigint, a number or a
```

Comment:
Be explicit about how a string is interpreted in this case.  I.e. the string is used as a set of raw bytes (in UTF-8, I believe) as input to key generation.  It's not interpreted as hex, or decimal, or anything else.

Also probably be explicit in saying the input can be any size, and all bytes will be used to generate entropy (vs. other inputs where a Buffer is expected to represent a number of a specific size).

Reply:
Same comment in all the other functions which take private key as input, so I won't duplicate this comment.

Reply:
Thanks! I updated them 👍🏽